src/components/CodeBlock/index.tsx

```diff
@@ -59,6 +65,7 @@ const getParams = (
 
   return {
     language: language.split("language-")[1],
+    languageFull: language || "language-plaintext",
```

Comment:
Is this used anywhere?

Reply:
nope! just a left over artifact from something i was testing.